### PR TITLE
Fix/3.0.x build2 pt2

### PR DESCRIPTION
**Pull request by mzedel**
closing:
- men-4855
- some bugs without a ticket + a regression in the deployments report, which I introduced in the previous fix PR 😞

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -52,22 +52,6 @@ cat >/var/www/mender-gui/dist/env.js <<EOF
   }
 EOF
 
-cat >/var/www/mender-gui/dist/livechat.js <<EOF
```
Comment (merlin-northern) [inline entrypoint.sh:55]: I verified, no left-overs:
```bash
[ fix/3.0.x-build2-pt2 ]
#grep -rniHF happy .
```

File: src/js/components/devices/authorized-devices.js [hunk 2]
```diff
@@ -78,15 +78,10 @@ export const Authorized = props => {
...
   useEffect(() => {
-    onSelectionChange([]);
     clearAllRetryTimers(setSnackbar);
     if (!filters.length && selectedGroup && groupFilters.length) {
       setDeviceFilters(groupFilters);
     }
-    clearInterval(timer);
-    // no group, no filters, all devices
-    timer = setInterval(getDevices, refreshDeviceLength);
-    getDevices();
```
Comment (merlin-northern) [inline src/js/components/devices/authorized-devices.js:94]: so now it will only get once when mounted, right?

Comment (mzedel) [reply src/js/components/devices/authorized-devices.js:94]: well, both effects/ trigger were run when mounted and the second trigger also is run on any relevant state change afterwards... This first one was basically a duplicate (based on the previous implementation, where these effects were split up) + relying only on the second one allows triggering it from a parent component...

**Review by merlin-northern — COMMENTED** [review]
lgtm, but I will defer to Fabio for his approval.

**Review by tranchitella — APPROVED** [review]

**Pull request merged**